api/v1alpha1/perconaservermysql_types.go

```diff
@@ -410,6 +410,11 @@ func (cr *PerconaServerMySQL) CheckNSetDefaults(serverVersion *platform.ServerVe
 		return errors.New("Orchestrator size must be 3 or greater and an odd number for raft setup")
 	}
 
+	if cr.Spec.Pause {
```

Comment:
Please add pause and allowUnsafeConfigurations to default cr.yaml if they should be supported:
This is how we have them in PXC operator:
```
  allowUnsafeConfigurations: false
#  pause: false
```

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/80/commits/bc1843afdc8568e862a948e5ac2e7afe1d6335cf